handlers/legacy.go

```diff
@@ -166,6 +166,7 @@ func (lh *Legacy) buildNotificationMessage(
 
 // HandleMessage handles a single AMQP delivery.
 func (lh *Legacy) HandleMessage(updateType string, delivery amqp.Delivery) error {
+	updateType = strings.ToLower(updateType)
```

Comment:
This is the primary change. The update type will always be converted to lower case.